skyportal/app_server.py

```diff
@@ -201,22 +204,26 @@ def log_request(self, handler):
     (r'/api/galaxy_catalog/ascii', GalaxyASCIIFileHandler),
     (r'/api/galaxy_catalog(/[0-9A-Za-z-_\.\+]+)?', GalaxyCatalogHandler),
     (
-        r'/api/(sources|spectra|gcn_event|shift)/([0-9A-Za-z-_\.\+]+)/comments',
+        r'/api/earthquake/([0-9A-Za-z-_\.\+]+)/ifo/([0-9A-Za-z-_\.\+]+)/predictions',
```

Comment:
what does `ifo` mean? Add a comment here?

Reply:
Clarified that this is an mmadetector.